README.md

```diff
@@ -337,6 +338,36 @@ in [Command Line Arguments](#command-line-arguments).
 you to start your application with SSL. More details in 
 [SSL Configuration](#ssl-configuration).
 
+## Back interface
+
+[Back](src/main/java/org/takes/http/Back.java) interface is the back-end that
+is responsible for accepting the incoming requests through the `Socket`, 
```

Comment:
@t-izbassar I think it's not correct, because HTTP requests and TCP sockets are on different network abstraction layers and only `BkBasic` is working with requests, I'd just say that `Back` interface is responsible for IO operations on TCP network layer.